lib/switchtec.c

```diff
@@ -774,7 +768,7 @@ static int read_log_defs(FILE *log_def_file, struct log_defs *defs)
 		tok = strtok(NULL, " \t");
 		if (!tok)
 			continue;
-        	mod_id = strtol(tok, NULL, 0);
+		mod_id = strtol(tok, NULL, 0);
```

Comment:
Always do whitespace cleanup in another patch.

Reply:
Sure.